src/demos/head.ts

```diff
@@ -0,0 +1,32 @@
+(() => {
```

Comment:
Fixme: This is an older version and missing some of the logic added to not load demo page reloader unless port is present and to not load the toggles for IE11.

https://github.com/Esri/calcite-app-components/blob/e99c18f00d98da1364e151b870dbf7b201fbaf8f/src/demos/head.js

Reply:
Good catch. Will fix.

Reply:
💥